### PR TITLE
Post via team repo ownership rather than who raised PR

### PR DESCRIPTION
**Pull request by MuriloDalRi**
## What

Change the seal behaviour to surface PRs raised against *repos owned by the team*, rather than its current behaviour of surfacing PRs raised by team members.

i.e. if a developer is in the Foo team, the Foo team is currently notified of all that developer's PRs regardless of which repos those PRs are in. Instead, we want the Seal to show PRs for repos owned by the team, so if the Foo team owns the Bar repo, it will be notified of all PRs to that repo regardless of who raised the PR.

## Why

Developers have agreed this will be useful - see [poll](https://gds.slack.com/archives/CAB4Q3QBW/p1660639759673839).

Some benefits:

1) It gives teams better visibility of their own repos
2) It prevents 'bulk PR noise', e.g. when a Platform Reliability dev raises Ruby upgrade PRs for repos outside of our team, we get reminded of those PRs constantly, even though it's up to the other teams to review/test/merge said PRs.

[Trello](https://trello.com/c/Pdh7bs59/2971-change-seal-to-post-via-team-repo-ownership-rather-than-who-raised-pr-3)

### CODE REVIEW

File: lib/github_fetcher.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
-require "octokit"
+require 'octokit'
 
 class GithubFetcher
   def initialize(team)
-    @organisation = ENV["SEAL_ORGANISATION"]
-    @github = Octokit::Client.new(access_token: ENV["GITHUB_TOKEN"])
-    @github.api_endpoint = ENV["GITHUB_API_ENDPOINT"] if ENV["GITHUB_API_ENDPOINT"]
+    @organisation = ENV['SEAL_ORGANISATION']
```
Comment (brucebolt) [inline lib/github_fetcher.rb]: Just a thing I noticed (haven't reviewed the rest of the PR, only just noticed this in passing): the changes here don't match with the rules configured in `rubocop-govuk` (e.g. we prefer double quotes over single quotes).  I think you're missing the `.rubocop.yml` configuration file.

Comment (MuriloDalRi) [reply lib/github_fetcher.rb]: Thanks, I'll change that!

Comment (MuriloDalRi) [reply lib/github_fetcher.rb]: I've added the govuk config and re-run Rubocop now

**Review by ollietreend — APPROVED** [review]
This looks good to me – really nice work @MuriloDalRi ✨🚀

I'm looking forward to this being released.

Also nice work with the other bits you did in this PR:

- Securing the quotes endpoint
- Removing the custom `Octocat.default_media_type`
- Adding Rubocop
- etc.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
 # Seal
 
-[![Build Status](https://travis-ci.org/binaryberry/seal.svg)](https://travis-ci.org/binaryberry/seal)
-
 ## What is it?
 
-This is a Slack bot that publishes a team's pull requests to their Slack Channel, once provided the organisation name and either a [Github team] (https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) name or the team members' individual github names. It is my first 20% project at GDS.
+This is a Slack bot that publishes a team's pull requests to their Slack Channel, once provided the organisation name, team names and respectivo repos.
```
Comment (catalinailie) [inline README.md]:
```suggestion
This is a Slack bot that publishes a team's pull requests to their Slack Channel, once provided the organisation name, team names and respective repos.
```

File: config/alphagov.yml [hunk 3]
```diff
@@ -273,136 +270,155 @@ govuk-platform-reliability:
...
+    - govwifi-logging-api
+    - govwifi-product-page
+    - govwifi-redirect
+    - govwifi-safe-restarter
+    - govwifi-shared-frontend
+    - govwifi-smoke-tests
+    - govwifi-tech-docs
+    - govwifi-terraform
+    - govwifi-user-signup-api
+    - govwifi-watchdog
```
Comment (catalinailie) [inline config/alphagov.yml]: This is missing a new line at the end of the file.

**Review by catalinailie — APPROVED** [review]
Looks good to me 👏  I have just a few small comments.

**Pull request merged**